src/index.html

```diff
@@ -7,12 +7,58 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="page">
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <img
+          src="images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+        />
```

Comment:
The `alt` attribute's value should be descriptive and present the content of the image if it fails to load. 'APPLE A1419 iMac 27' is not descriptive enough. Consider providing more details about the image for accessibility purposes, for example, 'APPLE A1419 iMac 27" on a white background'.